default.nix

```diff
@@ -1,7 +1,7 @@
 let
     ihp = builtins.fetchGit {
         url = "https://github.com/digitallyinduced/ihp.git";
-        ref = "refs/tags/v0.18.0";
+        rev = "03d6e232b0480cd3d7c3ab51f5b4e7e8b8cfab1f";
```

Comment:
I've just tagged the `v0.20.0` 👍 Could you switch this over to that tag?
https://github.com/digitallyinduced/ihp/releases/tag/v0.20.0